File: 프로그래머스/0/120830. 양꼬치/양꼬치.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

int solution(int n, int k) {
    int service = n / 10;                 // 서비스 음료수 개수
    int real_k = k - service;             // 실제 지불해야 하는 음료수 개수
    int answer = (n * 12000) + (real_k * 2000);
    return answer;
}

File: 프로그래머스/0/120830. 양꼬치/양꼬치.java
class Solution {
    public int solution(int n, int k) {
        int service = n / 10;             // 10인분당 1개의 서비스 음료
        int realK = k - service;          // 실제로 계산할 음료수 개수
        int answer = (n * 12000) + (realK * 2000);
        return answer;
    }
}
